CHANGELOG.md

```diff
@@ -16,11 +16,17 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 - Config parameter `external_address` is now a required value. (#826)
 
+- Config parameter `round_timeout` is removed.
+  `first_round_timeout` and `round_timeout_increase` should be used instead. (#848)
```

Comment:
I suggest to elaborate on it a little bit, just to make sure user will understand how to use new feature. And haven't you checked all the documentation? We need to update it too (perhaps in a different PRs)